docs/en/operation/Scalability-Tuning.md

```diff
@@ -8,20 +8,118 @@ priority: 9
 
 Alluxio is a scalable distributed file system designed to handle many workers within a single
 cluster.
-Several parameters can be tuned to prevent the Alluxio master from being overloaded.
-This page details the parameters to tune when scaling a cluster.
+This page details methods to recommend Alluxio node resource sizes. The section
+[Metrics to Monitor](#metrics-to-monitor) describes methods to monitor and estimate resources which
+influence the system's hardware requirements. The sections
+[Alluxio Master Configuration](#alluxio-master-configuration),
+[Alluxio Worker Configuration](#alluxio-worker-configuration), and
+[Alluxio Client Configuration](#alluxio-client-configuration) provide details on hardware and
+configuration tuning for large scale deployments.
 
 * Table of Contents
 {:toc}
 
+## Metrics To Monitor
+
+### Number of Files in Alluxio
```

Comment:
it will be better to mention what actions to take if a given metrics is really high.
e.g., for high number of files, either tuning higher heap size or switching to rocksdb

Reply:
The methods for tuning the hardware requirements are detailed in the `Master/Worker/Client Configuration` sections.  The impact of the metrics are detailed here in the second paragraph.